api-docs/graphql-api-introduction.md

```diff
@@ -51,3 +51,16 @@ The codeword differs for each preview field and you can find it by looking in th
 Our [GraphQL Explorer and Query builder](./graphql-explorer) is an interactive way to familiarize yourself with the available data models. Here you can easily delve into the underlying data sources that empower our GraphQL API, offering you a clearer understanding of the wealth of information at your fingertips.
 
 Additionally you can also download the [Public schema](https://apps.iris.trackunit.com/graphql-public-viewer/schema.gql) of the Trackunit Iris GraphQL API.
+
+## Advise for efficient querying with GraphQL
+
+When querying a GraphQL API, adopting certain practices can significantly enhance efficiency and responsiveness: 
+
+- Begin by precisely defining your data requirements using field selection, requesting only the information you genuinely need to reduce unnecessary data transfer. 
+- Leverage query fragments to reuse common selections across multiple queries, avoiding redundant code and promoting consistency. 
+- Utilize pagination techniques for fetching large sets of data, such as utilizing the `first`, `last`, `before`, and `after` arguments to manage the amount of data returned in each query.
```

Comment:
@mmt-trackunit Did we end up making 'first' or 'last' required?

Reply:
Yes. But some queries takes a list of IDs as well, and when they are given a list of IDs `first` or `last` is not required.